tests/test_upload.py

```diff
@@ -72,6 +73,27 @@ def test_successs_prints_release_urls(upload_settings, stub_repository, capsys):
     assert captured.out.count(NEW_RELEASE_URL) == 1
 
 
+def test_successs_prints_uploaded_package_names_if_verbose(upload_settings, capsys):
+    """Prints the path of each distribution attempting to be uploaded."""
+    dists_to_upload = [
+        helpers.WHEEL_FIXTURE,
+        helpers.SDIST_FIXTURE,
+        helpers.NEW_SDIST_FIXTURE,
+        helpers.NEW_WHEEL_FIXTURE,
+    ]
+
+    upload_settings.verbose = True
+
+    result = upload.upload(upload_settings, dists_to_upload)
+
+    assert result is None
+
+    captured = capsys.readouterr()
+
+    for dist_to_upload in dists_to_upload:
+        assert captured.out.count(dist_to_upload) == 1
```

Comment:
I think this test should also check for file size.

Reply:
I think this would be a good place to check for the fixture sizes that you've currently got in `test_get_file_size`.